kubemarine/patches/software_upgrade.yaml

```diff
@@ -8,8 +8,14 @@
 # The order of upgrade of defined by the implementation.
 
 thirdparties:
-  calicoctl: []
-  crictl: []
+  calicoctl:
+  - v1.27.13
+  - v1.28.9
+  - v1.29.4
+  crictl:
+  - v1.27.13
+  - v1.28.9
+  - v1.29.4
```

Comment:
These are new versions. There is nothing to upgrade for them

Reply:
updated